src/Symfony/Component/Notifier/Bridge/Bandwidth/BandwidthTransport.php

```diff
@@ -0,0 +1,171 @@
+<?php
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Bandwidth;
+
+use Symfony\Component\Notifier\Exception\InvalidArgumentException;
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Message\SmsMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author gnito-org <https://github.com/gnito-org>
+ */
+final class BandwidthTransport extends AbstractTransport
+{
+    protected const HOST = 'messaging.bandwidth.com';
+
+    private string $username;
+
+    private string $from;
+
+    private string $accountId;
+
+    private string $applicationId;
+
+    private ?string $priority;
+
+    private string $password;
+
+    public function __construct(
+        string $username,
+        #[\SensitiveParameter] string $password,
+        string $from,
+        string $accountId,
+        string $applicationId,
+        ?string $priority,
+        HttpClientInterface $client = null,
+        EventDispatcherInterface $dispatcher = null
+    )
+    {
+        $this->username = $username;
+        $this->password = $password;
+        $this->from = $from;
+        $this->accountId = $accountId;
+        $this->applicationId = $applicationId;
+        $this->priority = $priority;
+        parent::__construct(
+            $client,
+            $dispatcher
+        );
+    }
+
+    public function __toString(): string
+    {
+        return sprintf(
+            'bandwidth://%s?from=%s&account_id=%s&application_id=%s&priority=%s',
+            $this->getEndpoint(),
+            $this->from,
+            $this->accountId,
+            $this->applicationId,
+            $this->priority
```

Comment:
What if priority is null?

Reply:
The priority parameter will be omitted from the API call if null.